### DISCUSSION TITLE
堆叠模式递增开启后的问题

### DISCUSSION THREAD

**Discussion by paiap** [didi/LogicFlow#1316] [Q&A]
我开启了堆叠模式递增的功能，那么我应该如何去设置节点的置顶和置底呢？
使用toFront对于当前还是有问题（不是所有节点的最大zIndex+1的，似乎是1002开始？）

因为业务开始画了不少图了，肯定不能重画，我能拿到最大的zIndex值吗？
console.log(lf.graphModal.sortElements())
这个代码报错，我是最新的，是写法有问题么

**Comment by wumail** [comment]
报什么错啊

**Reply by paiap** [reply]
没有报错，就是开启递增后，新创建出来的节点并不是最高层的，不只是是什么问题，

**Reply by paiap** [reply]
如果有空，可以帮我看一下这个问题要有没有好的解决办法。
https://github.com/didi/LogicFlow/discussions/1307
